BREAKING_CHANGES.md

```diff
@@ -1,5 +1,8 @@
 # Breaking Changes
 
+## 2.1.0
```

Comment:
This isn't a breaking change.

Reply:
Think it was added like the other version changes.